### ISSUE TITLE
Country names output short alternatives when not requested

### ISSUE THREAD

**Issue by shaicoleman** [twitter/twitter-cldr-rb#221]
For example, in the file `common/main/pt.xml` the CLDR contains the following data
```
		<territories>
			<territory type="US">Estados Unidos</territory>
			<territory type="US" alt="short">EUA</territory>
		</territories>
```

However, the country name outputs the short alternative, when a short alternative wasn't requested. The expected behaviour would be to output the default variant.
```
TwitterCldr::Shared::Territories.from_territory_code_for_locale(:us, :pt)
=> "EUA"
TwitterCldr::Shared::Territories.from_territory_code_for_locale(:us, :id)
"A.S."
```
The acronyms might not be well known to all users around the world, and many times would not be easily searchable (e.g. try searching for EUA in google.pt)

Perhaps in the future an option could be added to those who wish to get the short variant, but it shouldn't be the default.

**Comment by camertron** [comment]
Hey @shaicoleman, thanks for reporting. I'm in the process of working on updating the library to CLDR v35.1 and hoping to have it ready by the end of the week. Do you have a sense for whether or not this particular problem is caused by relying on an outdated CLDR release, or is it a problem with the implementation?

**Comment by shaicoleman** [comment]
It seems to be caused by implementation.
I didn't look into the code, but my guess is that there is a hash of codes => names, and the hash entries get overwritten when they see the second entry with short name

**Comment by camertron** [comment]
@shaicoleman hmm ok. In that case, how would you feel about submitting a PR?

**Comment by shaicoleman** [comment]
@camertron, after running `rake update:locales_resources` it returns the full names

**Comment by camertron** [comment]
@shaicoleman oh nice, so updating to 35.1 or 36 should fix things for you?

**Comment by shaicoleman** [comment]
Yes, it should

**Comment by camertron** [comment]
Please see my comment on #222.

**Comment by shaicoleman** [comment]
`cldr35` branch looks good

**Comment by camertron** [comment]
Just published v5.0.0 of the gem :)
